Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml

```diff
@@ -97,6 +97,7 @@
       Female: FemaleVulpkanin
       Unsexed: MaleVulpkanin
   - type: DogVision
+  - type: Wagging
```

Comment:
Is that really the component name? Stupid.

Reply:
Real.